pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
+
         Return a new grouper with our resampler appended
+
+        Parameters
+        ----------
+        rule : str
+            The offset string or object representing target conversion.
+        *args
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
+        **kwargs
+            These parameters will be passed to the get_resampler_for_grouping
+            function.
+
+        Returns
+        -------
+        Grouper
+            Return a new grouper with our resampler appended
+
+        Examples
+        --------
+
+        Start by creating a DataFrame with 9 one minute timestamps.
+        >>> index = pd.date_range('1/1/2000', periods=9, freq='T')
```

Comment:
I'd say that we leave a blank line between the text and the code. Does it render all right in the html?

The convention is to use `idx` instead of `index`.